ui/node_menu.py

```diff
@@ -339,7 +360,10 @@ def draw(self, context):
         layout = self.layout
         insertNode(layout, "an_SoundBakeNode", "Bake Sound")
         insertNode(layout, "an_SoundFromSequencesNode", "Sound from Sequences")
+        insertNode(layout, "an_GetAllSequencesNode", "Get All Sequences")
+        insertNode(layout, "an_SequencesFromChannelNode", "from Channel")
```

Comment:
The name here shouldn't be "from Channel" in my opinion. The user maybe thinks that it means "Sound from Channel" which isn't the case. Maybe "Sequences from Channel" would be better. Or we make a new category for Sequence nodes